app_dart/lib/src/service/datastore.dart

```diff
@@ -35,10 +35,9 @@ typedef RetryHandler = Function();
 ///
 /// It uses quadratic backoff starting with 50ms and 3 max attempts.
 Future<void> runTransactionWithRetries(RetryHandler retryHandler,
-    {int delayMilliseconds = 50, int maxAttempts = 3}) {
-  final RetryOptions r = RetryOptions(
-      delayFactor: Duration(milliseconds: delayMilliseconds),
-      maxAttempts: maxAttempts);
+    {RetryOptions retryOptions}) {
+  final RetryOptions r =
```

Comment:
I would add a link to a GitHub issue that explains the rationale for this.

Reply:
Added the bug for context and added the rationale for the retries. Also kept the 3 retries only updates the backoff.